src/components/autocomplete-grid/autocomplete-grid-button/readme.md

```diff
@@ -0,0 +1,117 @@
+this means that the `sbb-autocomplete-grid-button` has a `button` role and its `id` is set based on the `sbb-autocomplete-grid-actions`'s `id`,
+which is needed to correctly set the `aria-activedescendant` on the related `input`.
+Moreover, the `sbb-autocomplete-grid-button` can't be focused via <kbd>Tab</kbd> due to the used pattern,
+since the focus must always stay on the connected `<input>`.
+
+<!-- Auto Generated Below -->
+
+## Properties
+
+| Name       | Attribute   | Privacy | Type                                       | Default    | Description                                                                                                                      |
+| ---------- | ----------- | ------- | ------------------------------------------ | ---------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `option`   | -           | public  | `SbbAutocompleteGridOptionElement \| null` |            | Gets the SbbAutocompleteGridOptionElement on the same row of the button.                                                         |
+| `disabled` | `disabled`  | public  | `boolean`                                  | `false`    | Whether the component is disabled.                                                                                               |
+| `negative` | `negative`  | public  | `boolean`                                  | `false`    | Negative coloring variant flag.                                                                                                  |
+| `iconName` | `icon-name` | public  | `string \| undefined`                      |            | The icon name we want to use, choose from the small icon variants from the ui-icons category from here https://icons.app.sbb.ch. |
+| `type`     | `type`      | public  | `SbbButtonType`                            | `'button'` | The type attribute to use for the button.                                                                                        |
+| `name`     | `name`      | public  | `string`                                   |            | The name of the button element.                                                                                                  |
+| `value`    | `value`     | public  | `string`                                   |            | The value of the button element.                                                                                                 |
+| `form`     | `form`      | public  | `string \| undefined`                      |            | The <form> element to associate the button with.                                                                                 |
+
+## Events
+
+| Name    | Type           | Description | Inherited From |
+| ------- | -------------- | ----------- | -------------- |
+| `click` | `PointerEvent` |             |                |
```

Comment:
I declared a public method named `dispatchClick` to handle the keyboard selection via `Enter` keypress.
The method has the `internal` annotation to not generate the doc, however these lines are still generated and i found no way to prevent it; 
the only possible change is to remove the `internal` annotation and display the docs, eg.

```
## Methods

| Name            | Privacy | Description                                                                                                             | Parameters             | Return | Inherited From |
| --------------- | ------- | ----------------------------------------------------------------------------------------------------------------------- | ---------------------- | ------ | -------------- |
| `dispatchClick` | public  | Used to dispatch a click event when users interact with the button via keyboard (the component does not receive focus). | `event: KeyboardEvent` | `void` |                |

```